### DISCUSSION TITLE
Jquery validate unhighlight and highlight callback equivalent

### DISCUSSION THREAD

**Discussion by doogdeb** [sinanbozkus/FormHelper#21] [Q&A]
In jqeury validate there are unhighlight and highlight callbacks where you can override the presentation of invalid fields. Is there an equivalent in FormHelper?

**Comment by sinanbozkus** [comment]
FormHelper doesn't support these methods. I am putting them on the to-do list. The new beta will have these callback methods.
Thank you for your great ideas.

**Reply by doogdeb** [reply]
No worries. 👍
